lib/paperclip_processors/mp3_paperclip_processor.rb

```diff
@@ -37,7 +37,7 @@ def copy_id3_tags_to_record(info)
         if value.respond_to?(:encode)
           value.encode(
             'UTF-8', invalid: :replace, undef: :replace, replace: "\ufffd"
-          ).mb_chars.normalize
+          ).unicode_normalize(:nfkc)
```

Comment:
@Manfred Since you recently added this, just wanted to tag you in case you had preferences.

```
DEPRECATION WARNING: ActiveSupport::Multibyte::Chars#normalize is deprecated and will be removed from Rails 6.1. Use #unicode_normalize(:nfkc) instead. (called from block in copy_id3_tags_to_record at /home/travis/build/sudara/alonetone/lib/paperclip_processors/mp3_paperclip_processor.rb:40)
```

Reply:
Oh god, the scourge of the bad Ruby Unicode APIs continues. Yeah, it's fine to replace it like that. Make sure you include the `:nfkc` argument because the default is `:nfc` and that's a really ignorant default.

Reply:
omg this conversation making me so happy :awwww: